### PR TITLE
[Improve]Remove http v1 code and disable mini load by default

### PR DESCRIPTION
**Pull request by hf200012**
1. http v2 has been actually tested in production, and it is completely replaceable to have http code. In order to simplify code maintenance, remove the previous http part of the code
2. The mini load function can completely pass the stream load function. In the next version, the mini load transition is disabled through the switch, and then this code is removed in the subsequent version. If users need to use the mini load function, they can add  in fe.conf `disable_mini_load=false` to enable, the default is disabled

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes)
2. Has unit tests been added: (No Need)
4. Has document been added or modified: (Yes)
5. Does it need to update dependencies: (No)
6. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/PaloFe.java [hunk 1]
```diff
@@ -134,24 +134,13 @@ public static void start(String dorisHomeDir, String pidDir, String[] args) {
...
-                httpServer2.setPort(Config.http_port);
-                httpServer2.setMaxHttpPostSize(Config.jetty_server_max_http_post_size);
-                httpServer2.setAcceptors(Config.jetty_server_acceptors);
-                httpServer2.setSelectors(Config.jetty_server_selectors);
-                httpServer2.setWorkers(Config.jetty_server_workers);
-                httpServer2.start(dorisHomeDir);
-            }
+            HttpServer httpServer2 = new HttpServer();
+            httpServer2.setPort(Config.http_port);
+            httpServer2.setMaxHttpPostSize(Config.jetty_server_max_http_post_size);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/PaloFe.java:139]: httpServer is ok

File: fe/fe-core/src/test/java/org/apache/doris/http/HttpAuthManagerTest.java [hunk 2]
```diff
@@ -31,15 +33,19 @@ public void testNormal() {
...
-        Assert.assertEquals(username, authMgr.getSessionValue(sessionId).currentUser.getQualifiedUser());
-        
+        List<String> sessionIds = new ArrayList<>();
+        sessionIds.add (sessionId);
+        System.out.println("username in test: " + authMgr.getSessionValue(sessionIds).currentUser);
+        Assert.assertEquals(username, authMgr.getSessionValue(sessionIds).currentUser.getQualifiedUser());
+
         String noExistSession = "no-exist-session-id";
-        Assert.assertNull(authMgr.getSessionValue(noExistSession));
+        sessionIds.clear ();
```
Comment (yiguolei) [inline fe/fe-core/src/test/java/org/apache/doris/http/HttpAuthManagerTest.java:46]: clear()

File: fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBDebugger.java [hunk 3]
```diff
@@ -85,21 +85,14 @@ public void startDebugMode(String dorisHomeDir) {
...
-        } else {
-            org.apache.doris.httpv2.HttpServer httpServer2 = new org.apache.doris.httpv2.HttpServer();
-            httpServer2.setPort(Config.http_port);
-            httpServer2.start(dorisHomeDir);
-        }
+
+        HttpServer httpServer = new HttpServer();
+        httpServer.setPort(Config.http_port);
+        httpServer.setMaxHttpPostSize(Config.jetty_server_max_http_post_size);
+        httpServer.setAcceptors(Config.jetty_server_acceptors);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBDebugger.java:92]: Why set these parameters but not set them in the past?

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java [hunk 4]
```diff
@@ -59,9 +62,14 @@
     @RequestMapping(path = "/api/{" + DB_KEY + "}/{" + TABLE_KEY + "}/_load", method = RequestMethod.PUT)
     public Object load(HttpServletRequest request, HttpServletResponse response,
                        @PathVariable(value = DB_KEY) String db, @PathVariable(value = TABLE_KEY) String table) {
-        this.isStreamLoad = false;
-        executeCheckPassword(request, response);
-        return executeWithoutPassword(request, response, db, table);
+        if(!Config.disable_mini_load) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java:65]: it is better to use if (condition) xxx not use if (!condition)

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 5]
```diff
@@ -1650,4 +1634,10 @@
      */
     @ConfField(mutable = true)
     public static boolean skip_compaction_slower_replica = true;
+
+    /**
+     * Mini load operation is disabled by default
+     */
+    public static boolean disable_mini_load = true;
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1641]: Missing @ConfField?

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by morningman — CHANGES_REQUESTED** [review]
Please don't put multiple types of changes into one PR.
Please split this PR into 3 parts:
1. disable mini load
2. remove http v1
3. modify http thread config

**Pull request closed without merging**